airflow/models/mappedoperator.py

```diff
@@ -470,7 +470,7 @@ def priority_weight(self) -> int:  # type: ignore[override]
         return self.partial_kwargs.get("priority_weight", DEFAULT_PRIORITY_WEIGHT)
 
     @property
-    def weight_rule(self) -> int:  # type: ignore[override]
+    def weight_rule(self) -> str:  # type: ignore[override]
```

Comment:
why do we need `type: ignore` here?

Reply:
We maybe not need it. Now, that we changed the return type from `int` to `str`, `MappedOperator.weight_rule` no longer deviates from it's parent `AbstractOperator.weight_rule` of type string.

Reply:
The `weight_rule ` is defined in the superclass (AbstractOperator) as a normal attribute, but here, we override it to a property, so Mypy doesn't like this override. We have the same ignore for all the overridden attributes.